src/blocks/blocks/posts/style.scss

```diff
@@ -64,6 +64,7 @@
 	--pag-cont-margin: 10px 0 0 0;
 
 	border: none;
+	max-width: inherit !important;
```

Comment:
One more clarification: Is this rule with `!important` still valid after you reverted to the previous solution? If only that filter were the problem, we could remove this change.

Reply:
@Soare-Robert-Daniel I've removed the `max-width` and it seems working fine.